### DISCUSSION TITLE
I cant see which SQL sql command was used to create database

### DISCUSSION THREAD

**Discussion by tananga** [supabase/supabase#14059] [Feature Requests]
Hello. I cant see which SQL command is executed after I click "Save" button in create table section. It would be great to see exactly what command is executed and confirm it. It would be a huge improvement and additional security for my newly created table.

![Screenshot 2023-04-29 at 01 39 12](https://user-images.githubusercontent.com/44244477/235270515-bd1200f1-1de6-46a2-91eb-d07a879b42d9.png)

Right now Im not able to see which SQL statement was execute after "save" is clicked and I would like to see a popup with SQL command which I can confirm or decline. Please tell me if it makes sense.

**Comment by gregnr** [comment]
Thanks for the suggestion @supabase-enjoyer (nice handle 😄)! I'm going to convert this into a discussion and we can keep the conversation going there!

**Comment by tananga** [comment]
It would be really great to see which SQL command is executed behind this nice UI. Also if something wrong is with UI nad lets say that checkbox is not working Im not able to see that it was not working.
